<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SodaMachine</class>
 <widget class="QMainWindow" name="SodaMachine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="b050_inserted">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>50ct inserted</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b100_inserted">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1â‚¬ inserted</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bAbort">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>abort</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bNoBottles">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>No bottles</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>65</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>User</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>65</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Machine</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>312</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
